src/api/constants/XLSX-CONFIG/index.ts

```diff
@@ -124,6 +125,16 @@ export const XLSX_ROW_INDEXES = (application: Application): XLSXRowIndexes => {
     indexes.TITLES.EXPORT_CONTRACT += 3;
   }
 
+  if (needPreCreditPeriodCover) {
+    indexes.BROKER_ADDRESS += 1;
+    indexes.BUYER_ADDRESS += 1;
+    indexes.LOSS_PAYEE_ADDRESS += 1;
+
+    indexes.TITLES.BUYER += 3;
+    indexes.TITLES.DECLARATIONS += 1;
+    indexes.TITLES.EXPORT_CONTRACT += 1;
+  }
+
```

Comment:
Did you run any tests and check the XLSX for any other scenarios? It looks like we also need to bump the index for `AGENT_ADDRESS`

Would be good to look at the XLSX with these changes, when fully populated, with a single and multiple policy